lib/vehicles_communicator.py

```
@@ -14,84 +16,95 @@ class VehiclesCommunicator:
-
-    def _get_telemetry(self, device):
-        return self._access_nested_dict(device, ["payload", "data", "telemetry", "position"])
 
-    def _send_request(self, url_postfix):
+    def _get_position(self, data):
         try:
-            response = requests.get(f"{self._url}{url_postfix}", params=self._params)
-            response.raise_for_status()
-            return response.json()
+            return data.get("telemetry").get("position")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)